tests/decorators/test_decorators.py

```diff
@@ -1411,3 +1411,168 @@ def dummy():
+
+        func = self._get_user_function(app)
+        assert_json(self, func, {
+            "scriptFile": "function_app.py",
+            "bindings": [
+                {
+                    "direction": BindingDirection.OUT,
+                    "type": TABLE,
+                    "name": "out",
+                    "rowKey": "dummy_key",
+                    "partitionKey": "dummy_partition_key",
+                    "tableName": "dummy_table_name",
+                    "connection": "dummy_out_conn"
+                },
+                {
+                    "direction": BindingDirection.IN,
+                    "type": TABLE,
+                    "name": "in",
+                    "rowKey": "dummy_key",
+                    "partitionKey": "dummy_partition_key",
+                    "tableName": "dummy_table_name",
+                    "connection": "dummy_in_conn",
+                },
+                {
+                    "direction": BindingDirection.IN,
+                    'dummyField': 'dummy',
+                    "type": HTTP_TRIGGER,
+                    "name": "trigger_name",
+                    "authLevel": AuthLevel.FUNCTION,
+                    "route": "dummy_route",
+                    "methods": [
+                        "GET", "PATCH"
+                    ]
+                },
+                {
+                    "direction": BindingDirection.OUT,
+                    'dummyField': 'dummy',
+                    "type": HTTP_OUTPUT,
+                    "name": "out",
+                }
+            ]
+        })
+
+    def test_table_input_binding(self):
+        app = self.func_app
+
+        @app.route()
+        @app.read_table(arg_name="in", table_name="dummy_table_name",
+                        connection="dummy_in_conn",
+                        row_key="dummy_key",
+                        partition_key="dummy_partition_key",
+                        data_type=DataType.STRING)
+        def dummy():
+            pass
+
+        func = self._get_user_function(app)
+
+        output = func.get_bindings()[0]
+
+        self.assertEqual(output.get_dict_repr(), {
+            "direction": BindingDirection.IN,
+            "dataType": DataType.STRING,
+            "type": TABLE,
+            "name": "in",
+            "tableName": "dummy_table_name",
+            "connection": "dummy_in_conn",
+            "rowKey": "dummy_key",
+            "partitionKey": "dummy_partition_key"
+
+        })
+
+    def test_table_output_binding(self):
+        app = self.func_app
+
+        @app.route()
+        @app.write_table(arg_name="out", table_name="dummy_table_name",
+                         connection="dummy_out_conn",
+                         row_key="dummy_key",
+                         partition_key="dummy_partition_key",
+                         data_type=DataType.STRING)
+        def dummy():
+            pass
+
+        func = self._get_user_function(app)
+
+        output = func.get_bindings()[0]
```

Comment:
do length check on get_bindings() before this line

Reply:
updated as per recommendation.